src/d3d12/d3d12-pipeline.cpp

```
@@ -202,6 +205,7 @@ Result createPipelineWithCache(
             pipelineCacheData = UnownedBlob::create(cachedBlob->GetBufferPointer(), cachedBlob->GetBufferSize());
             device->m_persistentPipelineCache->writeCache(pipelineCacheKey, pipelineCacheData);
             outCacheSize = pipelineCacheData->getBufferSize();
+            outCacheKey = pipelineCacheKey;
```

Cppcheck (2.21.0)

[style] The function 'makeFromDouble' is never used.

(unusedFunction)